solutions/arrays/lucky-number

```diff
@@ -0,0 +1,36 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+
+function findLuckyNumber(nums){
+  // implement this function
+  
+	let lastElement = -1, numberOfOccurences = 0;
+	for (let index = 0; index < nums.length; index++) {
+		if (nums[index] === lastElement) {
+			numberOfOccurences++;
+		}
+		else {
+			lastElement = nums[index];
+			numberOfOccurences = 1;
+		}
+		if (numberOfOccurences === lastElement)
```

Comment:
@DevratAgarwal This solution will fail for `1 1 2`. Expected output is `-1`, where as this solution will give `1`.